<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionViewerWidget</class>
 <widget class="QWidget" name="ActionViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActionViewerWidget</string>
  </property>
  <widget class="QListWidget" name="modelNameListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="actionListWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="17" column="1">
     <widget class="QCheckBox" name="showCenterModelOctreeBox">
      <property name="text">
       <string>Show Center Model Octree</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="showPredictedSkelButton">
      <property name="text">
       <string>Show Predicted Skels</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="showSampledSkelButton">
      <property name="text">
       <string>Show Sampled Skels</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLabel" name="sliderSkelNumLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QCheckBox" name="showSampleRangeBox">
      <property name="text">
       <string>Show Sampling Range</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="showEndPoseBox">
      <property name="text">
       <string>Show End Pose</string>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QCheckBox" name="showCenterModelVoxelBox">
      <property name="text">
       <string>Show Center Model Voxel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="showStartPoseBox">
      <property name="text">
       <string>Show Start Pose</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="showPredictedSkelNumLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSlider" name="showSkelNumSlider">
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="showSampledSkelNumLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="refreshSkelButton">
      <property name="text">
       <string>Refresh Skeletons</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="classProbThresholdEdit">
      <property name="text">
       <string>0.8</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QPushButton" name="repredictingButton">
      <property name="text">
       <string>Re-predicting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
